invenio_app_ils/search/api.py

```diff
@@ -37,7 +38,7 @@ def search_by_document_pid(self, document_pid=None, filter_states=None,
         if document_pid:
             search = search.filter("term", document_pid=document_pid)
         else:
-            raise ValueError("Must specify document_pid param")
+            raise PropertyRequired(description="document_pid is required")
```

Comment:
In this case it would be `RequiredParameterMissingError` not property. Sorry for being a naming police.
Also we search in items so the exception class should not be imported from circulation IMHO.

Reply:
Same as above, circulation stuff.